### PR TITLE
Priorityq

### PR DESCRIPTION
**Pull request by mastokley**

### CODE REVIEW

File: src/test_priorityq.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+
+def test_insert_on_empty():
+    from priorityq import PriorityQueue
+    pq = PriorityQueue()
+    pq.insert((7, "banana"))
+    assert pq._container == [(7, "banana")]
+
+
+@pytest.mark.parametrize(("populated_pq", "insert_item", "result_pq"), INSERT_PARAMETERS)
```
Comment (joelstanner) [inline src/test_priorityq.py:27]: Avoid long lines over 80 characters (for this class). We want to adhere to PEP8. You can continue on the next line with implicit line joining:

``` python
@pytest.mark.parametrize(("populated_pq", "insert_item", "result_pq"),
                         INSERT_PARAMETERS)
```

File: src/test_priorityq.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    pq.peek()
+    assert pq._container == result_pq
+
+
+@pytest.mark.parametrize(("populated_pq", "insert_item", "result_pq"), INSERT_PARAMETERS)
+def test_peek(populated_pq, insert_item, result_pq):
+    from priorityq import PriorityQueue
+    pq = PriorityQueue()
+    pq._container = result_pq
+    assert pq.peek() == result_pq[0]
```
Comment (joelstanner) [inline src/test_priorityq.py:59]: Make sure to add tests for edge cases such as inserting, peeking, and popping from a queue with 1 or zero items, and show that your data structure behaves as it should in those cases.

File: src/priorityq.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        else:
+            raise IndexError("Priority queue empty")
+
+    def peek(self):
+        """Return highest priority item in priority queue without removing it."""
+        return self._container[0]
+        pass
+
+    def _sort_down(self, index):
+        # helper method to maintain heap property after popping
```
Comment (joelstanner) [inline src/priorityq.py:34]: Be consistent with your docstrings. The triple quote style used on the other functions is preferred, even if these are just helper methods. Reserve `#` comments for block or line level comments.
